eslintrc.js

```diff
@@ -9,9 +9,14 @@ module.exports = {
 
   rules: {
     camelcase: ['error', { properties: 'always' }],
+    'no-debugger': ['warn'],
     'no-var': ['error'],
     'prefer-const': ['error'],
     semi: ['error', 'always'],
+    'sort-keys': ['error', 'asc', {
+      'caseSensitive': false,
+      'natural': false
+    }]
```

Comment:
Also, just being completely picky, would you remove the quotations from the object keys